src/objective-c/GRPCClient/version.h

```diff
@@ -1,6 +1,6 @@
 /*
  *
- * Copyright 2015 gRPC authors.
+ * Copyright 2019 gRPC authors.
```

Comment:
Seems like the file was just renamed. I guess that doesn't require a copyright date change?

Reply:
Either way makes sense to me actually... but changing back since I do have a slight preference to leaving the year the same on a moved file. Thanks for pointing it out.